res/TensorFlowPythonExamples/examples/unique/__init__.py

```diff
@@ -0,0 +1,4 @@
+import tensorflow as tf
+
+in_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(1, 6), name="unique_in")
+op_ = tf.compat.v1.math.argmin(in_)
```

Comment:
this needs fix

Reply:
Yes, seems like a manual error. I will fix it in the next commit. Thanks again for spotting.